en/coding-the-legislation/periods.md

```diff
@@ -1,162 +0,0 @@
-# Periods
-
-> For a better understanding of this section please read first the [periods](periods.md) definition  
-
-
-Every variable formulas needs to be defined over a specific period or needs values of other variables on a specific period.  
-
-This section is dedicated to a full comprehension of coding the time in OpenFisca.
-
-###### Table of Contents :
-   
-[The  `periods` module](https://doc.openfisca.fr/en/coding-the-legislation/periods.html#the-module-periods) : how to **convert time** in OpenFisca object      
-    
-[Calling periods in variable definition](https://doc.openfisca.fr/en/coding-the-legislation/periods.html#calling-periods-in-variable-definition): how to **code variable** with calls to convenient period
-### The `periods` module
```

Comment:
This is useful, but more a reference documentation about `periods` than an actual guide to write formulas.

Should be integrated later in the reference doc.